### PR TITLE
Wordpress

### LINKED ISSUE #6: Implement Wordpress.com backend using API connection for editing copy
This is fairly large of a task.  We want to create a Wordpress.com purely for editing the website's content.  The website will remain in React, but will consume a Wordpress API.  Before attempting this, please volunteer to do so in this thread and let's all discuss the technical implementations of such a PR.

### PR DESCRIPTION
**Pull request by cereallarceny**
This effectively closes #6.  Merging this PR **requires us** to do the deploy the Wordpress back-end that has been (so far) created locally.  Once this is deployed, we could in theory review, merge, and redeploy.  Redeployment of this fix is 100% a breaking change - here's how this process should be done to minimize website downtime:

1. Deploy Wordpress back-end (pending @cereallarceny)
2. Change production link in this PR.
3. Review and merge this PR to `master`
4. Deploy `master`
5. Deploy changes on openmined-serverless that also haven't been committed yet (pending @cereallarceny)
6. Make new issues based on `TODO`'s marked in the source code

What this does not yet accomplish is creating a blog.  We have, tentatively speaking, set this up.  But no styles have been created for displaying blog posts.  I will consider that a separate branch and PR altogether, but it's on its way @iamtrask.

### CODE REVIEW

**Comment by cereallarceny** [comment]
In layman's terms...

This PR basically removes all written content (and functional content for that matter) from the front-end codebase and replaces it with data retrieved from a Wordpress back-end deployed to our private AWS account.

File: src/modules/index.js [hunk 1]
```diff
@@ -3,10 +3,24 @@ import { routerReducer } from 'react-router-redux';
...
+export const SERVERLESS_API_URL =
+  process.env.NODE_ENV === 'development'
+    ? 'http://localhost:3001'
+    : 'https://yltw3rj1r3.execute-api.us-east-1.amazonaws.com/dev';
+
+// NOTE: Make sure you're running the openmined-serverless application alongside
+export const WORDPRESS_API_URL =
+  process.env.NODE_ENV === 'development'
+    ? 'http://localhost:8080/wp-json'
+    : 'http://localhost:8080/wp-json'; // TODO: We don't yet have a deployed version of the Wordpress API
```
Comment (kevinahuber) [inline src/modules/index.js]: we do now :) 

It'd be great to have all of these URL's in `.env` files instead of depending on `NODE_ENV`, but not a dealbreaker

**Comment by cereallarceny** [comment]
@kevinahuber This is ready to go once we deploy the API on SSL and then we'll change the link in this PR.  From there, let's merge and start the deploy.

**Comment by cereallarceny** [comment]
Closing in favor of #16
